### ISSUE TITLE
[Bug]: Tank accumulated consumption stopped updating

### ISSUE THREAD

**Issue by agnarvold** [cjaliaga/home-assistant-aquarea#117]
### The problem

The entity for «tank accumulated consumption» stopped updating three days ago (15.8.24). The total «accumulated consumption» and «heating accumulated consumption» is working. I have tried to run the reinstall from HACS and reload the integration without success. Also tried to restart and update home assistant. Anyone experienced the same? Fix?

### What version of Home Assistant Core are you running?

2024.8.2

### What type of installation are you running?

Home Assistant Core

### What version of the integration are you running?

0.7.0

### Device description

1 zone - a2w heat pump with dhw

### Is the integration loading?

Yes

### Is the adapter able to connect to the Aquarea Smart Cloud with a stable internet connection?

Yes

### Are you able to perform the action you're trying to with the Aquarea Smart Cloud Site/App?

Yes

### Area of the issue

Tank accumulated consumption

**Comment by lbasek** [comment]
Yes, I also have the same problem. The problem started on August 11th for some reason.
@agnarvold @cjaliaga

**Comment by cjaliaga** [comment]
Thanks for reporting this. Is the data available in the aquarea site for you folks?

**Comment by lbasek** [comment]
Yes, I can see the correct data within the Panasonic application.

**Comment by agnarvold** [comment]
Same here, the data is being logged as usual in the panasonic system.
![IMG_4981](https://github.com/user-attachments/assets/a0a53245-2fbd-4750-b846-09412a044658)

**Comment by afharo** [comment]
I'm experiencing a similar issue: I get some tank readings, but overall, I'm missing metrics, which results in a consumption of around 50% of the actually reported via the wall control and the Panasonic app. This happens for both: the tank and the cooling.

If I plot it in a histogram, I notice that the bars are more separated than before (in a 48h graph, it used to show a bar every hour, and now there are blanks in many of those hours):
<img width="1465" alt="image" src="https://github.com/user-attachments/assets/1f2be153-9d46-4283-8563-8289769ae0b9">
_👆 each bar represents the `change` in the accumulated consumption.

I wonder if the integration is failing to collect some hours and not retrying, hence missing that info.

Happy to create a separate issue if you think that it requires a separate investigation.

**Comment by jakublipinski** [comment]
I am experiencing the same issue.
I checked the Panasonic API and it provides the consumption numbers correctly.
In my case all the consumption entities were disabled in Home Assistant for some reason and I had to reenable them manually. I will keep checking if the consumption numbers are reported.

**Comment by vukisz** [comment]
Same here. It's reporting but very unreliably. Panasonic comfort cloud app shows 2.99 kW for today while tank accumulated consumption increased by 0.5 kw today
